01-Basic/01-variables.js

```
@@ -37,4 +37,144 @@ var persona = {
+
+function greet() {
+  console.log(greeting);
+}
+
+greet()
+
+
+
+//> Ámbito local o de función
+function greet() {
+    let greeting = "Hello World";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'greet'. Consider to delete it or rename it.

'greet' is defined here:


(lint/suspicious/noRedeclare)